[Content] Coalition: Lunarium Intro (#6646)
endless-sky/endless-sky: data/coalition/coalition outfits.txt

> 	description "In their early days of space exploration, the Saryds developed the fuel module to travel far from Saros without worrying about becoming stranded without fuel, or sudden fuel leaks."

Reviewer: Perhaps `without worrying about running out of fuel or sudden fuel leaks`

> 	description "In their early days of space exploration, the Saryds developed the fuel module to travel far from Saros without worrying about becoming stranded without fuel, or sudden fuel leaks."

Reviewer 2: > Perhaps `without worrying about running out of fuel or sudden fuel leaks`

I would simplify that further into `without worrying about running out of fuel or sudden leaks`

	description "	Nowadays, nearly every system in the Coalition has refuelling facilities, but some captains still find use in the extra jumps these modules offer."

outfit "Decoy Plating"
	category "Systems"
	cost 93000
	thumbnail "outfit/decoy plating"
	"mass" 7
	"outfit space" -7
	"scan interference" 2
	"radar jamming" 5

> 	description "This plating takes advantage of a ship's scan logs, and reproduces the data of other ships nearby, effectively deflecting attempts to scan a ship's true cargo and equipment."

Reviewer 3: ```suggestion
	description "This plating takes advantage of a ship's scan logs and reproduces the data of other ships nearby, effectively deflecting attempts to scan a ship's true cargo and equipment."
```

> 	description "This plating takes advantage of a ship's scan logs, and reproduces the data of other ships it nearby, effectively deflecting attempts to scan a ship's true cargo and equipment."

Reviewer 4: ```suggestion
	description "This plating takes advantage of a ship's scan logs, and reproduces the data of other ships it is close to, effectively deflecting attempts to scan a ship's true cargo and equipment."
```

	description "	Out of a need to protect themselves against the destructive Heliarch torpedoes, the Lunarium also incorporated small, independent radar jamming systems in each section of the plating, if only to act as a last line of defense."

outfit "Refuelling Module"
	category "Systems"
	cost 822000
	thumbnail "outfit/refuelling module"
	"mass" 10
	"outfit space" -10
	"fuel capacity" -100
	"fuel generation" .1
	"energy consumption" .02

> 	description "To avoid the risk of being inspected by the Heliarch as they refuel on planets, Lunarium captains employ this outfit when carrying supplies or stolen outfits."

Reviewer 3: ```suggestion
	description "To avoid the risk of being inspected by the Heliarchs as they refuel on planets, Lunarium captains employ this outfit when carrying supplies or stolen outfits."
```

	description "	The outfit's mechanisms need a partitioned amount of fuel set aside to allow it to work properly, so a ship can only install so many of these before becoming incapable of jumping to another system."

outfit "Shield Refactor Module"
	category "Systems"
	cost 2843000
	thumbnail "outfit/shield refactor module"
	"mass" 7
	"outfit space" -7
	"hull repair rate" -.2
	"shield energy" .2
	"shield heat" .24
	"shield generation multiplier" .15
	"hull repair multiplier" -.2
	description "In theory, all shield generators can operate at a much faster rate than normal, if pushed to their true limit. However, the constant need to reform shields during a heated battle quickly overloads shield generators and damages them in the process. By relocating dedicated hull repair systems to focus exclusively on repairing such damage, this compact device allows shield generators to reach their true potential."

> 	description "	A much needed boon against the Heliarch torpedoes' devastating effects on hull plating, the Lunarium Government highly favors production of this expensive piece over other shield generators due to its effectiveness in preventing shields from being taken down."

Author: I'll drop the word government.
